CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,31 @@
+[Code of Conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md)
+
+Percona Community is a community around developing, installing, running, optimizing, learning everything around databases and the needed infrastructure.
+
+## The ABCs
+
+* **A**ssume good intentions.
+* **B**e nice to each other - being civil and respectful goes a long way.
+* **C**riticize code, tools, other types of contributions, and situations - not people and their personalities.
+
+## Rules
+
+We respect all members of the Percona Community - that involves everyone’s contributions in code, text, visuals, and chat. Your contributions might take place in one of our GitHub organizations, on Jira, on the forums, Slack, or elsewhere.
+
+We all are committed to participating in a community where abuse and harrassment is neither tolerated nor accepted. If you want to give feedback, remember to criticize the code or other type of contribution, not the person.
```

Comment:
```suggestion
We all are committed to participating in a community where abuse and harassment is neither tolerated nor accepted.
```
Part of it is a duplicate of ABCs.

Reply:
Could you please update the source of this one? I just would copy paste from thee the new version: https://github.com/percona/community/blob/main/content/contribute/coc.md